.travis.yml

```diff
@@ -0,0 +1,15 @@
+language: csharp
+solution: MoreLinq.sln
+mono: latest
+dist: trusty
+sudo: required
+dotnet: 1.0.0-preview2-003131
+
+install:
+  - dotnet restore
+
+script:
+  - dotnet build -c Debug MoreLinq
```

Comment:
> I could add both if you want to.

I'd say it's even more important to test release but if it's not too much trouble to have both then we should strive for it.

> `dotnet` insisted on trying to use the debug build.

Really? 😮 I guess we could leave it as a separate issue to be investigated?